### PR TITLE
feat(Firestore): add sample for multiple inequality filters

### PR DESCRIPTION
**Pull request by bshaffer**
- adds sample `query_filter_compound_multi_ineq`
- Removes now invalid samples `query_filter_range_invalid` and `query_order_field_invalid`

### CODE REVIEW

File: firestore/src/query_filter_compound_multi_ineq.php [hunk 1]
```diff
@@ -26,25 +26,36 @@
...
-    $invalidRangeQuery->documents();
+    # [START firestore_query_filter_compound_multi_ineq]
+    $chainedQuery = $collection
+        ->where('age', '>', 35)
+        ->where('height', '>', 60)
+        ->where('height', '<', 70);
+    # [END firestore_query_filter_compound_multi_ineq]
+    foreach ($chainedQuery->documents() as $document) {
+        printf(
+            'Document %s returned by age > 35 and heigiht between 60 and 70' . PHP_EOL,
```
Comment (bshaffer) [inline firestore/src/query_filter_compound_multi_ineq.php:55]:
```suggestion
            'Document %s returned by age > 35 and height between 60 and 70' . PHP_EOL,
```

File: firestore/test/firestoreTest.php [hunk 2]
```diff
@@ -275,6 +274,12 @@ public function testChainedQuery()
         $this->assertStringContainsString('Document SF returned by query state=CA and name=San Francisco', $output);
     }
 
+    public function testChainedInequalityQuery()
+    {
+        $output = $this->runFirestoreSnippet('query_filter_compound_multi_ineq');
+        $this->assertStringContainsString('Document person4 returned by age > 35 and heigiht between 60 and 70', $output);
```
Comment (bshaffer) [inline firestore/test/firestoreTest.php:280]:
```suggestion
        $this->assertStringContainsString('Document person4 returned by age > 35 and height between 60 and 70', $output);
```

**Review by bshaffer — COMMENTED** [review]
typos!

**Comment by clairekeer1997** [comment]
Hi Brent, this is Jia from Firestore queries team. Thanks for adding the examples for PHP! The example for Datastore looks good. But for FIrestore, this example query 
```$chainedQuery = $citiesRef
        ->where('born', '>', 1900)
        ->where('born', '<', 1950);
```
only contains inequalities on a single field `born`. We need to have inequalities on different fields to be considered as a multiple inequalities query. For example, `age > 20 AND height > 60` is a query with multiple inequalities.

**Comment by bshaffer** [comment]
@clairekeer1997 hi Claire. If you click the "Files Changed", you can see the actual sample. What you're referring to is not part of this sample, which looks like this:

```
    $chainedQuery = $collection
        ->where('age', '>', 35)
        ->where('height', '>', 60)
        ->where('height', '<', 70);
```

**Comment by clairekeer1997** [comment]
Sorry I read on the wrong one. The updated example looks good to me.

**Pull request merged**